hazelcast/src/main/java/com/hazelcast/spi/discovery/impl/DefaultDiscoveryService.java

```diff
@@ -98,6 +98,28 @@ public void destroy() {
         }
     }
 
+    @Override
+    public void markEndpointAsUnhealthy(Address address) {
+        for (DiscoveryStrategy discoveryStrategy : discoveryStrategies) {
+            discoveryStrategy.markEndpointAsUnhealthy(address);
+        }
+    }
+
+    @Override
+    public Set<Address> getUnhealthyEndpoints() {
+        Set<Address> addresses = null;
+        for (DiscoveryStrategy strategy : discoveryStrategies) {
+            Set<Address> local = strategy.getUnhealthyEndpoints();
```

Comment:
nit: "Address" type and "local" as variable name made me think this is the "local member address" and I started wondering "why do we care about local address here?"... maybe my brain is wired the wrong way, but can you think of a better name for this variable?

Reply:
Good catch @vbekiaris, thanks - I've renamed both variables in this block to clarify their usage: https://github.com/hazelcast/hazelcast/pull/25364/commits/de3c107ceee89f0a859d916eed9269082f84c3ac